integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -14,7 +14,7 @@
 
   // domain constants
   public static final String DOMAIN_VERSION = Optional.ofNullable(System.getenv("DOMAIN_VERSION"))
-      .orElse("v8");
+      .orElse("v9");
```

Comment:
why not use the constant in integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java rather than changing this to keep in sync?

Reply:
I can look at this, but my thought process was that any place that was previously defaulting to "v8" would likely want to default now to "v9" unless it was explicitly an upgrade test.

Reply:
I made the other codepoint depend on this one.